### PR TITLE
Added samples for ContentDialog

### PR DESCRIPTION
**Pull request by mathieuwilsonnv**
3 samples that showcase different properties

### CODE REVIEW

File: src/Uno.Playground.Shared/Samples/ContentDialog.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		{
+			var result = await new Uno.Playground.Styles.Controls.ContentDialogRateApp().ShowAsync();
+		}
+
+		private async void ShowTermsDialog(object sender, object args)
+		{
+			var result = await new Uno.Playground.Styles.Controls.ContentDialog_TermsAndConditions().ShowAsync();
+		}
+
+		private void ContentDialog_PrimaryButtonClick(object sender, ContentDialogButtonClickEventArgs args)
```
Comment (MatFillion) [inline src/Uno.Playground.Shared/Samples/ContentDialog.xaml.cs]: What are those empty methods for?

File: src/Uno.Playground.Shared/Styles/Controls/ContentDialogRateApp.xaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    Title="Do you like our app?"
+    PrimaryButtonText="I love it!"
+    SecondaryButtonText="It's good"
+	CloseButtonText="Close"
+    PrimaryButtonClick="ContentDialog_PrimaryButtonClick"
+    SecondaryButtonClick="ContentDialog_SecondaryButtonClick"
+	CloseButtonClick="ContentDialog_CloseButtonClick"
+	IsSecondaryButtonEnabled="False">
+
+	<!--Want to replace with the Uno logo but it doesn't want to show-->
```
Comment (MatFillion) [inline src/Uno.Playground.Shared/Styles/Controls/ContentDialogRateApp.xaml]: Is this comment necessary?

Comment (mathieuwilsonnv) [reply src/Uno.Playground.Shared/Styles/Controls/ContentDialogRateApp.xaml]: Definitely not

File: src/Uno.Playground.Shared/Samples/ContentDialog.xaml.cs (hunk 1 above)
Comment (mathieuwilsonnv) [reply src/Uno.Playground.Shared/Samples/ContentDialog.xaml.cs]: Not necessary

**Comment by jeromelaban** [comment]
Not needed anymore

**Pull request closed without merging**